api/src/main/java/com/datatorrent/api/Module.java

```diff
@@ -1,17 +1,20 @@
 /**
- * Copyright (C) 2015 DataTorrent, Inc.
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This was already fixed on the feature-module branch. Please pull the latest changes into your fork.